addons/website_slides_survey/models/slide_slide.py

```diff
@@ -65,3 +64,27 @@ def _action_set_viewed(self, target_partner, quiz_attempts_inc=False):
                         'slide_partner_id': new_slide_partner.id
                     }
                 )
+
+    def action_get_slide_survey_urls(self):
+        if not all(slide.channel_id.is_member for slide in self):
+            return None
+        return self._action_get_slide_survey_urls()
+
+    def _action_get_slide_survey_urls(self):
+        certification_infos = []
```

Comment:
should be a dict where the key the slide id, and the value is another dict with the url, survey_id, certif_done, ...